sdk/src/lib.rs

```diff
@@ -16,7 +16,7 @@ pub use env::RawVal;
 pub use env::Status;
 pub use env::Symbol;
 pub use env::TryFromVal;
-use env::*;
+pub use env::*;
```

Comment:
Instead of pubing the env to expose the `Env` trait, if you merge the main branch into this PR you should find that the `Env` type in the SDK is a struct that you can add methods to, and then we should add some methods there for accessing contract data and unwrapping/wrapping types that are convertible to and from RawVal.

Reply:
Fixed, thanks.